src/components/inputManager.js

```diff
@@ -214,53 +204,9 @@ define(['playbackManager', 'focusManager', 'appRouter', 'dom'], function (playba
             case 'toggledisplaymirror':
                 playbackManager.toggleDisplayMirroring();
                 break;
-            case 'togglestats':
-                //playbackManager.toggleStats();
-                break;
-            case 'movies':
-                // TODO
-                appRouter.goHome();
```

Comment:
Why remove these that call `goHome`?

Reply:
I removed those three because it didn't seem like they served their intended purpose, which I interpreted as to go to a section with music. We haven't documented the available commands yet so my plan was to just not include these three.